src/Rules/Resources/Descriptions.resx

```diff
@@ -160,7 +160,7 @@
     <value>The IsKeyboardFocusable property for the given element is false for an element where it would normally be true. However, the IsOffscreen property on the element is true, so the value of IsKeyboardFocusable may be acceptable.</value>
   </data>
   <data name="IsKeyboardFocusableForCustomShouldBeTrue" xml:space="preserve">
-    <value>The IsKeyboardFocusable property for a custome element should be true when the element supports actionable patterns.</value>
+    <value>The IsKeyboardFocusable property for a custom element should be true when the element supports actionable patterns.</value>
```

Comment:
There should be a corresponding change in `Descriptions.Designer.cs`. These are automatically kept in sync if edited via the IDE, but require manual synchronization if the files are edited outside the IDE.